File: caesar.go
package main

import (
	"bufio"
	"os"
	"strings"
)

func main() {
	alphabet := "abcdefghijklmnopqrstuvwxyz"
	//shift by 5
	cipher := "fghijklmnopqrstuvwxyzabcde"

	reader := bufio.NewReader(os.Stdin)
	println("What would you like to encrypt?")
	inputString, _ := reader.ReadString('\n')

	var encrypted []string
	for i := 0; i < len(inputString); i++ {
		position := strings.Index(alphabet, string(inputString[i]))
		newChar := string(cipher[position])
		//println(newChar)
		encrypted = append(encrypted, newChar)
	}
	println(strings.Join(encrypted, ""))
}

File: caesar.lua
alphabet = "abcdefghijklmnopqrstuvwxyz"
--shift by 5
cipher = "fghijklmnopqrstuvwxyzabcde"

io.write("What would you lke to encrypt?\n")
input_string=io.read()
encrypted = ''
for c in input_string:gmatch"." do
    position = string.find(alphabet, c)
    new_char = cipher:sub(position,position)
    encrypted = encrypted .. new_char
end
print("Encrypted string: ")
print(encrypted)
